### PR TITLE
Add portage-git container with a shallow ::gentoo git

### PR DESCRIPTION
**Pull request by Flowdalic**

### CODE REVIEW

File: portage-git.Dockerfile [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+# sync friendly ::gentoo git repository from the official sources and
+# verifies its checksum as well as its gpg signature.
+
+FROM --platform=$BUILDPLATFORM alpine:3.11 as builder
+
+ARG GIT_REPO="https://github.com/gentoo-mirror/gentoo.git"
+ARG SIGNING_KEY="F748E9B3C47E393CC24C8FAF7C2AC09CD98F2EDF"
+
+RUN apk add --no-cache ca-certificates gnupg git
+RUN	git clone --depth 1 "${GIT_REPO}" /gentoo-git
```
Comment (robbat2) [inline portage-git.Dockerfile:13]: is this extra space intentional?

Comment (Flowdalic) [reply portage-git.Dockerfile]: No, that was a stray tab character. Thanks for pointing it out.

**Comment by ajakk** [comment]
My last comments on this in IRC:

```
19:35 <+ajak> Flow: i do think we'd not be gaining much by giving people another way to use ::gentoo in OCI containers, yeah
19:36 <+ajak> and if such a repository is ever meant to be pulled, then people will need to remember to set the volatile flag, and i'd prefer to not give people a footgun to use a "shallow" repository that actually grows over time
19:37 <+ajak> (of course i recall having a tidy solution for that problem, but alas..)
```

**Comment by Flowdalic** [comment]
Users wouldn't need to set the volatile flag, just "mounting" the container under /var/db/repos would do the right thing.

I personally think that the advantages of having a git-sync-friendly ::gentoo container outweigh its disadvantages. Especially since we already had a user asking for it in #gentoo-containers.

That said, if you feel strongly about it, then feel free to close this PR. Having it open for months without any feedback about it is not ideal.

**Comment by ajakk** [comment]
> Having it open for months without any feedback about it is not ideal.

Well, I guess I thought you'd see pings on IRC where there was active discussion, but maybe not. I note there isn't a description here either ;)

We already maintain an image suitable for use as the Gentoo repository, so I don't see the need for a Git equivalent.

**Comment by Flowdalic** [comment]
> Well, I guess I thought you'd see pings on IRC where there was active discussion, but maybe not.

IRC discussions are not a substitution for PR actions. I wonder why this wasn't closed months ago.

> I note there isn't a description here either

That seems unrelated, the PR title is pretty self explaining.

**Pull request closed without merging**